packages/cli/src/lib/setup/setupMultihost.ts

```diff
@@ -254,6 +286,7 @@ function Multihost(options) {
             });
 
             rl.question(query, value => {
+                // @ts-expect-error what is this for?
```

Comment:
most likely to erase the password from the readline interface history (this property exists internally).
I'm not sure it carries over to the actual shell though.

Reply:
At least I cannot access what I entered in a readline interface outside of it. Probably can be removed.